app/views/foreman_webhooks/payload_templates/webhook_payload_-_payload_default.erb

```diff
@@ -0,0 +1,11 @@
+<%#
+name: Webhook Payload - Payload Default
+description: This template is used to define default content of payload for a webhook.
+snippet: false
+model: PayloadTemplate
+%>
+<%=
+as_json({
+  id: @object.id
+})
+%>
```

Comment:
This is quite weird, you essentially create JSON-like Ruby syntax and then let Ruby to convert it :-)

I think what's more readable, safer and user-priendly is just to use ERB to generate the JSON.

Reply:
Okay, might be actually useful. Let's see this in practice.